platform/platform-app/pom.xml

```diff
@@ -323,13 +323,13 @@
         DO NOT EDIT UNLESS UPDATING CXF-->
         <!-- Copied from apache-cxf-3.1.4-src/parent/pom.xml -->
         <cxf.abdera.version>1.1.3</cxf.abdera.version>
-        <cxf.activemq.version>5.12.0</cxf.activemq.version>
+        <cxf.activemq.version>5.12.2</cxf.activemq.version>
         <cxf.ahc.version>1.9.8</cxf.ahc.version>
         <cxf.apacheds.version>2.0.0-M23</cxf.apacheds.version>
         <cxf.atmosphere.version>2.3.7</cxf.atmosphere.version>
-        <cxf.atmosphere.version.range>[2.0,3.0)</cxf.atmosphere.version.range>
+        <cxf.atmosphere.version.range>[2.0,2.4)</cxf.atmosphere.version.range>
         <cxf.axiom.version>1.2.14</cxf.axiom.version>
-        <cxf.bcprov.version>1.52</cxf.bcprov.version>
+        <cxf.bcprov.version>1.55</cxf.bcprov.version>
```

Comment:
This property isn't used in `platform/platform-app/src/main/resources/features.xml`, so it shouldn't need to be updated here.